scripts/rmc_flags.py

```diff
@@ -4,15 +4,57 @@
 
 import argparse
 
+# Taken from https://github.com/python/cpython/blob/3.9/Lib/argparse.py#L858
+# Cannot use `BooleanOptionalAction` with Python 3.8
```

Comment:
Is there an issue with upgrading to Python 3.9? I know there has been some chat about Python versions lately.

Reply:
Only to reduce the burden on RMC users.